lib/driver.js

```diff
@@ -199,23 +204,6 @@ class WebDriverAgentDriver extends BaseDriver {
     return sim;
   }
 
-  async launchApp () {
```

Comment:
This was an accident, I will fix and refactor once we work through the rest of the questions.